pkg/code-generator/schemagen/crd.go

```diff
@@ -0,0 +1,123 @@
+package schemagen
+
+import (
+	"bufio"
+	"bytes"
+	"fmt"
+	"log"
+	"os"
+	"path/filepath"
+	"strings"
+
+	"github.com/rotisserie/eris"
+
+	code_generator "github.com/solo-io/solo-kit/pkg/code-generator"
+	"github.com/solo-io/solo-kit/pkg/code-generator/writer"
+	"k8s.io/apiextensions-apiserver/pkg/apis/apiextensions/v1beta1"
+	"k8s.io/utils/pointer"
+
+	"github.com/ghodss/yaml"
+	kubeyaml "k8s.io/apimachinery/pkg/util/yaml"
+)
+
+const (
+	apiVersion = "apiextensions.k8s.io/v1beta1"
```

Comment:
rename to v1beta1? might even be able to grab this constant from a k8s dependency instead